engine/Computation/TableMatrix/CreateTableMatrix.py

```diff
@@ -40,4 +38,5 @@ def create_matrix(num_of_var: int) -> list:
 
     logger.debug("In create_matrix(): new_matrix=%s", new_matrix)
     logger.info("Finished creating the initial 'truth' matrix.")
+    logger.info("*** Initial Matrix:\t%s ***", new_matrix)
```

Comment:
Is this a candidate for tabular or the new tabbed print function? Meaning to have the matrix printed/logged more like a matrix than in a single line?

Reply:
Yes. It will be added with another ticket.